### PR TITLE
LPS-121893 Migrate v2 management toolbar usages in notifications-web

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
**Test plan:**
1. Login with Test user
2. Create a new user
3. Go to D&M and share an item with the new user
4. Logout and login with the new user
5. On top right, click on the user icon and then "Notifications"

Screenshot 2021-02-22 at 12 33 30

/cc @markocikos @julien @jonmak08

### CODE REVIEW

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Review by julien — APPROVED** [review]

File: modules/apps/notifications/notifications-web/src/main/resources/META-INF/resources/notifications/view.jsp [hunk 1]
```diff
@@ -67,14 +67,27 @@ navigationURL.setParameter(SearchContainer.DEFAULT_CUR_PARAM, "0");
 	%>'
 />
 
-<clay:management-toolbar-v2
+<portlet:actionURL name="deleteNotifications" var="deleteNotificationsURL"><portlet:param name="redirect" value="<%= currentURL %>" /></portlet:actionURL>
+<portlet:actionURL name="markNotificationsAsRead" var="markNotificationsAsReadURL"><portlet:param name="redirect" value="<%= currentURL %>" /></portlet:actionURL>
+<portlet:actionURL name="markNotificationsAsUnread" var="markNotificationsAsUnreadURL"><portlet:param name="redirect" value="<%= currentURL %>" /></portlet:actionURL>
```
[the remark this answers is not in the document]
Comment (ambrinchaudhary) [reply modules/apps/notifications/notifications-web/src/main/resources/META-INF/resources/notifications/view.jsp]: Changed! Thanks @markocikos

**Comment by boton** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Review by carloslancha — APPROVED** [review]

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
@boton tested and working, LGTM, feel free to forward it. Thx!

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia4.giphy.com%2Fmedia%2Fta83CqOoRwfwQ%2Fgiphy.gif&token=shipit" data-gitmeme-token="shipit"></a>

**Comment by boton** [comment]
Sorry about the delay :(

**Comment by boton** [comment]
ci:forward

**Review by boton — APPROVED** [review]

**Pull request closed without merging**